ehrql/query_language.py

```diff
@@ -386,6 +386,10 @@ def is_between(self, start, end):
     def is_on_or_between(self, start, end):
         return (self >= start) & (self <= end)
 
+    def is_during(self, interval):
+        start, end = interval
+        return self.is_on_or_between(start, end)
```

Comment:
Reluctantly, I agree. I wonder if, at some point, we could do something clever so that where we have multiple spec tests which produce identical query model we only run them once against real databases? But that's something to tackle as and when the speed of the spec tests becomes problematic.